pkgs/tools/virtualization/cloud-init/default.nix

```diff
@@ -25,6 +28,13 @@ python3.pkgs.buildPythonApplication rec {
   prePatch = ''
     substituteInPlace setup.py \
       --replace /lib/systemd $out/lib/systemd
+
+    substituteInPlace cloudinit/net/networkd.py \
+      --replace "['/usr/sbin', '/bin']" "['/usr/sbin', '/bin', '${iproute2}/bin', '${pkgs.systemd}/bin']"
```

Comment:
```suggestion
      --replace "['/usr/sbin', '/bin']" "['/usr/sbin', '/bin', '${iproute2}/bin', '${systemd}/bin']"
```

Reply:
Tried this, will not work. `systemd` refers to `python3Packages.systemd`. Is there a better way to do this?

Reply:
inherit pkgs.systemd in python-packages.